pkgs/development/libraries/fmt/default.nix

```
@@ -3,29 +3,41 @@
 stdenv.mkDerivation rec {
   version = "5.2.1";
   name = "fmt-${version}";
+
   src = fetchFromGitHub {
     owner = "fmtlib";
     repo = "fmt";
     rev = "${version}";
     sha256 = "1cd8yq8va457iir1hlf17ksx11fx2hlb8i4jml8gj1875pizm0pk";
   };
-  nativeBuildInputs = [ cmake ];
+
+  outputs = [ "out" "dev" ];
+
+  buildInputs = [ cmake ];
```

Why have you moved `cmake` from `nativeBuildInputs` to `buildInputs`?
Shouldn't `cmake` be a *build time only* dependency?

```suggestion
  nativeBuildInputs = [ cmake ];
```

---

Yes indeed. cmake belongs in `nativeBuildInputs`

---

Ups, that move was a mistake. Fixed.